### PR TITLE
Add some settings from 4.12 to angular

### LINKED ISSUE #245: Add new features to Ng for v4.1.2
**Is your feature request related to a problem? Please describe.**

Go through change log for 4.12 in ids enterprise https://github.com/infor-design/enterprise/blob/4.12.x/docs/CHANGELOG.md

Add any new settings or make issues for the more complicated things that cannot be hit on first pass (which will be listed here)

- [x] #963 - Add new settings for message dialog
- [x] #473 - Input for clear formatting button
- [x] #889 - useCurrentTime setting #261
- [x] #378 - Added export separator option
- [x] #667 - Add ability to run code on click on empty message
- [x] #250 - Standlone Paging

### PR DESCRIPTION
**Pull request by tmcconechy**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Added some settings and features that were added in 4.12.
- Add new settings for message dialog
- Input for clear formatting button
- useCurrentTime setting #261
- Added export separator option
- Add ability to run code on click on empty message

Bumped test version to 4.13-rc2

**Related github/jira issue (required)**:
Closes #245

**Steps necessary to review your pull request (required)**:
http://localhost:4200/message
- added new buttons to test different alerts

http://localhost:4200/editor
- added clear formatting button

http://localhost:4200/datepicker
- added currentTime example

http://localhost:4200/datagrid-export-without-datagrid
- tested the CSV export button

http://localhost:4200/emptymessage
- added a button, which fires code (an alert)

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,11 @@
     - `@types/jquery` has been updated to 3.3.21.
 - `[App]` Added a service to control renderLoop. A global timer used by enterprise to control animations. See index.html for usage example. See app.component.ts for example on how to manually start. ([#214](https://github.com/infor-design/enterprise-ng/issues/214)) `KOH`
 - `[General]` Added Breakpoints typedef for ids-enterprise breakpoints.js api ([#258](https://github.com/infor-design/enterprise-ng/issues/258))`PWP`
+- `[Message]` Add new settings for message dialog (alert/info type) ([#245](https://github.com/infor-design/enterprise-ng/issues/245)) `TJM`
+- `[Editor]` Add new icon for clear formatting button in the editor ([#245](https://github.com/infor-design/enterprise-ng/issues/245)) `TJM`
+- `[Datepicker]` Added useCurrentTime setting ([#245](https://github.com/infor-design/enterprise-ng/issues/245)) `TJM`
+- `[Export]` Added seperator option ([#245](https://github.com/infor-design/enterprise-ng/issues/245)) `TJM`
```
Comment (fitzorama) [inline docs/CHANGELOG.md]: separator

File: projects/ids-enterprise-ng/src/lib/datagrid/soho-datagrid.component.ts [hunk 2]
```diff
@@ -1525,10 +1525,11 @@ export class SohoDataGridComponent implements OnInit, AfterViewInit, OnDestroy,
    * Trigger export of grid data to CSV formatted file.
    * @param fileName The prefix name to be used for the exported file.
    * @param customDs A datasource to override the default.
+   * @param separator The seperator to use in the cvs file, defaults to 'sep=,'
```
Comment (fitzorama) [inline projects/ids-enterprise-ng/src/lib/datagrid/soho-datagrid.component.ts]: separator

File: projects/ids-enterprise-ng/src/lib/utils/soho-utils.d.ts [hunk 3]
```diff
@@ -8,10 +8,11 @@ interface SohoUtilsExcelStatic {
    * Export the grid contents to csv
    * @param fileName The desired export filename in the download.
    * @param customDs An optional customized version of the data to use.
+   * @param separator The seperator to use in the cvs file, defaults to 'sep=,'
```
Comment (fitzorama) [inline projects/ids-enterprise-ng/src/lib/utils/soho-utils.d.ts]: separator

File: projects/ids-enterprise-ng/package.json [hunk 4]
```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@types/jquery": "3.3.21",
     "@types/d3": "4.13.0",
-    "ids-enterprise": "4.13.0-beta.0",
+    "ids-enterprise": "4.13.0-rc.2",
```
Comment (pwpatton) [inline projects/ids-enterprise-ng/package.json]: Could set to 4.13.0

**Pull request merged**